peachjam/models/judgment.py

```diff
@@ -72,10 +72,31 @@ def __str__(self):
         return self.name
 
 
+class CourtRegistry(models.Model):
+    court = models.ForeignKey(
```

Comment:
We'll need a code on this model. I suggest we make it `f'{self.court.code}-{slugify(self.name)}` which will be unique, and which we can set on save. That will be necessary for import/export support.